### PR TITLE
5814 mpm5

### PR DESCRIPTION
**Pull request by dmuelle**
#5814

### CODE REVIEW

File: modules/ROOT/pages/introduction-monitoring-metrics.adoc [hunk 1]
```diff
@@ -105,54 +109,63 @@ However, the MicroProfile Metrics feature doesn't provide metrics about Open Lib
 
 If you enable both the MicroProfile Metrics and Performance Monitoring features, then the MXBeans for monitoring and the `/metrics` REST endpoint are activated.
 In this case, metrics for Open Liberty server components are exposed through both interfaces.
-The MicroProfile Metrics feature version 2.3 and later automatically enables the Performance Monitoring feature.
+The MicroProfile Metrics feature versions 2.3 and later automatically enable the Performance Monitoring feature.
 
-==  Integration with Micrometer
+[#micrometer]
+== Integration with Micrometer
```
Comment (donbourne) [inline modules/ROOT/pages/introduction-monitoring-metrics.adoc:115]: the list of micrometer integrations changes over time.  would be better not to specifically say "18"  monitoring systems (I currently count 19 on the page we link to).

File: modules/ROOT/pages/introduction-monitoring-metrics.adoc [hunk 2]
```diff
@@ -105,54 +109,63 @@ However, the MicroProfile Metrics feature doesn't provide metrics about Open Lib
...
 
 The feature:mpMetrics-5.0[display=MicroProfile Metrics 5.0] feature integrates with the Micrometer metrics technology to transfer metric data to multiple third-party monitoring systems. Micrometer enables you to instrument your code in a standardized way to visualize metrics results in the monitoring system of your choice. Micrometer can connect to 18 different third-party metric monitoring systems. For a full list of metric monitoring systems, see the https://micrometer.io/docs[Micrometer metrics documentation].  
 
-You can integrate Open Liberty with Micrometer by providing the Micrometer core library, Micrometer registry implementations, and the related dependencies to the Open Liberty runtime. This configuration strategy does not provide access to the Micrometer metrics API in the application.
+You can integrate Open Liberty with Micrometer by completing the following steps.
```
Comment (donbourne) [inline modules/ROOT/pages/introduction-monitoring-metrics.adoc:119]: You can integrate Open Liberty with one or more of the monitoring systems that work with Micrometer by completing the following steps.

File: modules/ROOT/pages/introduction-monitoring-metrics.adoc [hunk 3]
```diff
@@ -105,54 +109,63 @@ However, the MicroProfile Metrics feature doesn't provide metrics about Open Lib
...
 
-This configuration applies to all Micrometer registry implementations except for the Prometheus registry implementation, which is enabled by default when the Prometheus registry library and its dependencies are available. The `<registry>` value is an identifying value that is designated for each individual Micrometer registry. For the respective values for each Micrometer registry, see  the https://micrometer.io/docs[Micrometer metrics documentation].
+3. Use MicroProfile Config properties for each Micrometer registry to configure the connection to the associated monitoring tool. 
++
+These properties are documented in xref:microprofile-config-properties.adoc#metrics[MicroProfile Config properties: MicroProfile Metrics]. They properties use the following the syntax: `mp.metrics.<registry>.<property>`,  where `<registry>` is the identifying value that is defined for that registry by Micrometer. For example, properties for the Elastic registry follow the  `mp.metrics.elastic.<property>` syntax. For information about the properties that are available for each registry, see the Micrometer documentation.
```
Comment (donbourne) [inline modules/ROOT/pages/introduction-monitoring-metrics.adoc:133]: They properties -> The properties
use the following the -> use the following

File: modules/ROOT/pages/microservice-observability-metrics.adoc [hunk 4]
```diff
@@ -55,82 +57,112 @@ public String countingThings() {
...
 
-Use the `@Counted` annotation to mark a method, constructor, or type as a counter.
-The counter increments monotonically, counting total invocations of the annotated method:
+@Counted(name="methodCounter", tags={"method=bar"})
+public void bar() {
+    //bar something
+}
+----
+
+When you re-use the same metric name, the set of tags must having matching set of tag names.
```
Comment (donbourne) [inline modules/ROOT/pages/microservice-observability-metrics.adoc:87]: must having matching set of tag names -> must have a matching set of tag names

File: modules/ROOT/pages/microservice-observability-metrics.adoc [hunk 5]
```diff
@@ -189,105 +225,9 @@ donationDistribution_Dollars_sum{mp_scope="application",tier="integration",} 918
 donationDistribution_Dollars_max{mp_scope="application",tier="integration",} 1000.0
 ----
 
-'''
-
-[#concurrentgaugeOld]
-=== Concurrent gauge [MicroProfile Metrics 2.0-4.0]
-A concurrent gauge metric counts the concurrent invocations of an annotated element.
-This metric also tracks the high and low watermarks of each invocation.
+Starting in MicroProfile Metrics 5.0, you can adjust the the percentile precision of the `Histogram` metrics by using the `mp.metrics.smallrye.histogram.precision` MicroProfile Config property. The property accepts a value from 1 to 5 and is defaulted to 3 if no value is specified. A greater value results in more exact percentile calculations, but at a greater memory cost. For more information, see xref:microprofile-config-properties.adoc#metrics[MicroProfile Config properties: MicroProfile Metrics].
```
Comment (donbourne) [inline modules/ROOT/pages/microservice-observability-metrics.adoc:228]: the the -> the

File: modules/ROOT/pages/microservice-observability-metrics.adoc [hunk 6]
```diff
@@ -55,82 +57,112 @@ public String countingThings() {
...
+public String donateAmountViaCreditCard(@FormParam("amount") Long amount, @FormParam("card") String card) {
+
+    if (processCard(card, amount))
+        return "Thanks for donating!";
+
+    return "Sorry, please try again.";
+}
+----
+
+Starting in MicroProfile Metrics 5.0, you can adjust the the percentile precision of the `Timer` metrics by using the `mp.metrics.smallrye.timer.precision` MicroProfile Config property. The property accepts a value from 1 to 5 and is defaulted to 3 if no value is specified. A greater value results in more exact percentile calculations, but at a greater memory cost. For more information, see xref:microprofile-config-properties.adoc#metrics[MicroProfile Config properties: MicroProfile Metrics].
```
Comment (donbourne) [inline modules/ROOT/pages/microservice-observability-metrics.adoc:156]: the the -> the

**Review by donbourne — COMMENTED** [review]
A few comments on things I noticed in this PR...

**Comment by dmuelle** [comment]
Thanks @donbourne - these edits are addressed in #6236

**Pull request merged**